avogadro/qtplugins/atompairbonds/atompairbonds.h

```diff
@@ -27,11 +27,11 @@ class AtomPairBonds : public QtGui::ScenePlugin
   void process(const QtGui::Molecule& molecule,
                Rendering::GroupNode& node) override;
 
-  QString name() const override { return tr("Atom-Pair Bonds"); }
+  QString name() const override { return tr("Non-Covalent Bonds"); }
```

Comment:
Let's just stick with "Non-Covalent" and leave off the bonds. Lots of debates about what is a bond and what is not a bond. 😄